Source/WTF/wtf/text/Base64.cpp

```diff
@@ -263,10 +263,9 @@ std::optional<Vector<uint8_t>> base64Decode(std::span<const std::byte> input, Ba
 
 std::optional<Vector<uint8_t>> base64Decode(StringView input, Base64DecodeMode mode)
 {
-    unsigned length = input.length();
-    if (!length || input.is8Bit())
-        return base64DecodeInternal(std::span(input.characters8(), length), mode);
-    return base64DecodeInternal(std::span(input.characters16(), length), mode);
+    if (input.isEmpty() || input.is8Bit())
```

Comment:
I think this only needs to check `is8Bit`. I don’t think `isEmpty` leads to additional optimization or is needed for correctness.